setup.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 
-import vulcan
 from setuptools import setup
 
 setup(name='saltstack-vulcan',
-    version=vulcan.__version__,
+    version='0.1.2',
```

Comment:
this ends up with the version defined in two places and deviates from the Python convention of putting it into __version__. For example.

```
>>> import requests
>>> requests.__version__
'2.13.0'
```